release/rllib_tests/learning_tests/hard_learning_tests.yaml

```diff
@@ -311,20 +312,21 @@ dqn-breakoutnoframeskip-v4:
         dueling: false
         num_atoms: 1
         noisy: false
-        prioritized_replay: false
+        replay_buffer_config:
+          type: MultiAgentReplayBuffer
+          learning_starts: 20000
+          capacity: 1000000
+          prioritized_replay_alpha: 0.5
```

Comment:
Also here, comment, why we are specifying `prioritized_replay_alpha` if we don't even use the prio-replay buffer.
Probably save to remove the `prioritized_replay_alpha` altogether.

Reply:
Ok, I thought it was there hinting "if you use PER, use it with this alpha!".
Removing it!